services/streammanager/kafka/kafkamanager.go

```diff
@@ -611,7 +617,11 @@ func sendMessage(ctx context.Context, jsonData json.RawMessage, p producerManage
 			return makeErrorResponse(fmt.Errorf("unable to serialize event with messageId: %s, with error %s", messageId, err))
 		}
 	}
+	if config.MultiTopicSupport {
+		topic = parsedJSON.Get("topic").Value().(string)
```

Comment:
type cast will panic here if the field is absent or not a string. Additionally, shouldn't we fallback to the default topic if the message doesn't contain one?

Reply:
Handled similar vulnerabilities